src/components/responsive-toggle/index.js

```diff
@@ -6,20 +6,21 @@
  import { __ } from '@wordpress/i18n'
  import { useState, useCallback } from '@wordpress/element'
  import { dispatch } from '@wordpress/data'
- 
+ import getUAGEditorStateLocalStorage from '@Controls/getUAGEditorStateLocalStorage';
```

Comment:
Extra white space before all lines

```suggestion
import getUAGEditorStateLocalStorage from '@Controls/getUAGEditorStateLocalStorage';
```

Reply:
Removed